HelpSource/Classes/ProxySpace.schelp

```diff
@@ -170,6 +170,21 @@ a boolean.
 argument::includeSettings
 a boolean.
 
+method::asCode
+
+if proxyspace is globally accessible, this posts a code string that can access it.
```

Comment:
--> If the proxyspace...

Reply:
19 methods in this helpfile begin the explanation with lowercase, 3 with uppercase ...
 is there a general style rule about this?

Reply:
we don't need style rules for general rules of english grammar; it should be capitalized.

Reply:
fine with me, I'll fix it here. 
Upper/lowercase is completely mixed across all helpfile method lines, 
but why not start the Island of Perfect Grammar here ;-)